shardy/dialect/sdy/ir/attrs.td

```diff
@@ -697,6 +745,10 @@ def Sdy_DimMapping : AttrDef<Sdy_Dialect, "DimMapping"> {
 def Sdy_TensorMapping : AttrDef<Sdy_Dialect, "TensorMapping"> {
   let mnemonic = "tensor_mapping";
   let summary = "Factor mappings for each dimension of a tensor.";
+  let description = [{
+      **Constraints:**
+      - Elements in `dim_mappings` must satisfy the constraints in `DimMappingAttr`.
```

Comment:
Sorry I mislead you, `No duplicate factor indices across dimensions` should also be added here :)

Reply:
Oh! got it - thanks :D